docs/installation.rst

```diff
@@ -13,12 +13,23 @@ Requirements
 - The ``--container-runtime`` option must correspond to the containerization tool you use.
 - ``ansible-builder`` version ``3.x`` requires Python ``3.9`` or higher.
 
+.. note::
+
+   The **recommended** approach to install ``ansible-builder`` is using the ``ansible-dev-tools`` package.
```

Comment:
Please remove recommended and use something similar to the text in this PR. https://github.com/ansible/ansible-builder/pull/662